src/crypto-api.ts

```diff
@@ -490,5 +501,15 @@ export interface GeneratedSecretStorageKey {
     encodedPrivateKey?: string;
 }
 
+/**
+ * Options object for {@link CryptoApi#checkOwnCrossSigningTrust}.
+ */
+export interface CheckOwnCrossSigningTrustOpts {
+    /**
+     * If true, will fetch the private cross signing keys from the secret storage
+     */
```

Comment:
what happens if it is false?

Reply:
The cross signing keys are not imported

Reply:
ok, please could you make this explicit in the comment.